src/Stripe/Entities/StripeCountrySpec.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Stripe.Infrastructure;
+
+namespace Stripe
+{
+    public class StripeCountrySpec : StripeObject
+    {
+        [JsonProperty("object")]
+        public string Object { get; set; }
+
+        [JsonProperty("livemode")]
+        public bool LiveMode { get; set; }
+
+        [JsonProperty("default_currency")]
+        public string DefaultCurrency { get; set; }
+
+        [JsonProperty("supported_bank_account_currencies")]
+        public Dictionary<string, IEnumerable<string>> SupportedBankAccountCurrencies { get; set; }
```

Comment:
the dictionary is good, but List<string> is probably more intuitive for the second argument. It all comes back at once anyway.